config/kubernetes-incubator/org.yaml

```diff
@@ -65,6 +65,7 @@ members:
 - fgrzadkowski
 - flyingcougar
 - foxish
+- girikuncoro
```

Comment:
We are moving to deprecate the kubernetes-incubator org (https://github.com/kubernetes/community/issues/1922), so we want to hold off adding new members to the org.

If there is any particular repo that you'd like to contribute to in the incubator org, could you or any other OWNERS create an issue for moving the repo to kubernetes-sigs like https://github.com/kubernetes/org/issues/1094?

Reply:
thanks for the heads up @nikhita. actually, most of the repos I'm contributing to is in the kubernetes-sigs org, I'll take my name out from the kubernetes-incubator